mcp-plugin.py

```diff
@@ -723,6 +723,26 @@ def set_comment(
             return
         cfunc.del_orphan_cmts()
         cfunc.save_user_cmts()
+    raise IDAError(f"Failed to set comment at {address}")

```

Comment:
This check was removed on purpose. The set_comment will set on the disassembly first and then do a best effort for the decompilation.